content/docs/languages/csharp/dotnet.md

```diff
@@ -2,11 +2,11 @@
 title: gRPC for .NET
 ---
 
-For an introduction to gRPC for .NET ([grpc-dotnet][]) and a basic tutorial, see
-the following pages:
+The following pages cover the gRPC for .NET implementation:
```

Comment:
`for the .NET implementation`

Reply:
Yeah, that text is a bit off. I've reworked it. PTAL